src/operations/property-filter.ts

```diff
@@ -59,6 +59,11 @@ const filterUsingOperator = (
       return (itemValue + '').toLowerCase().indexOf((tokenValue + '').toLowerCase()) > -1;
     case '!:':
       return (itemValue + '').toLowerCase().indexOf((tokenValue + '').toLowerCase()) === -1;
+    case '^':
+      return (itemValue + '').toLowerCase().startsWith((tokenValue + '').toLowerCase());
+    // The unsupported operators result in no data being filtered out.
+    default:
+      return true;
```

Comment:
Not sure about this one. Should we throw a runtime error instead?

Because people may start relying on non-operator chars for their custom cases, and we will never be able to add new operators https://www.hyrumslaw.com/